geometry_msgs/PoseStamped meas_world
float32 yaw
